recipes/default.rb

```diff
@@ -40,4 +40,79 @@
     user 'root'
     group 'root'
     mode 0500
-end
\ No newline at end of file
+end
+
+if node['install']['cloud'].casecmp?("aws")
+  
+  remote_file "#{node['cloud']['init']['install_dir']}/awscliv2.zip" do
+    source node['cloud']['init']['awscli']['url']
+    user 'root'
+    group 'root'
+    mode 0500
+    action :create
+  end
+
+  package "unzip"
+  
+  bash "unzip and install AWS CLI V2" do
+      user 'root'
+      group 'root'
+      code <<-EOH
+          set -e
+          rm -f #{node['cloud']['init']['install_dir']}/aws
+          unzip #{node['cloud']['init']['install_dir']}/awscliv2.zip
+          ./aws/install
+          rm #{node['cloud']['init']['install_dir']}/awscliv2.zip
+      EOH
+  end
+
+  directory node['cloud']['init']['docker']['config_dir'] do
+    owner "root"
+    group "root"
+    mode 0500
+    action :create
+  end
+
+  template "#{node['cloud']['init']['docker']['config_dir']}/config.json" do
+      source "config.json.erb"
+      user 'root'
+      group 'root'
+      mode 0500
+  end
+
+  directory "/home/glassfish/.docker" do
```

Comment:
There is an attribute for glassfish home https://github.com/logicalclocks/hopsworks-chef/blob/master/attributes/default.rb#L31

Reply:
cloud-chef is not dependent on hopsworks-chef to use this attribute. The only solution is to move this attribute to conda-chef and then use it in hopsworks-chef and cloud-chef